docs/sampling.asciidoc

```diff
@@ -37,37 +87,25 @@ This means the following data will always accurately reflect *all* of your appli
 * Transaction breakdown metrics
 * Errors, error occurrence, and error rate
 
-// To turn off the sending of all data, including transaction and error data, set `active` to `false`.
-
 [float]
-==== Sample rates
+=== Sample rates
 
 What's the best sampling rate? Unfortunately, there isn't one.
 Sampling is dependent on your data, the throughput of your application, data retainment policies, and other factors.
 A sampling rate from `.1%` to `100%` would all be considered normal.
-You may even decide to have a unique sample rate per service -- for example, if a certain service
-experiences considerably more or less traffic than another.
+You'll likely decide on a unique sample rate for different scenarios.
+Here are some examples:
 
-// Regardless, cost conscious customers are likely to be fine with a lower sample rate.
+* Services with considerably more traffic than others might be safe to sample at lower rates
+* Routes that are more important than others might be sampled at higher rates
+* A production service environment might warrant a higher sampling rate than a development environment
 
-[float]
-==== Sampling with distributed tracing
-
-The initiating service makes the sampling decision in a distributed trace,
-and all downstream services respect that decision.
-
-In each example below, `Service A` initiates four transactions.
-In the first example, `Service A` samples at `.5` (`50%`). In the second, `Service A` samples at `1` (`100%`).
-Each subsequent service respects the initial sampling decision, regardless of their configured sample rate.
-The result is a sampling percentage that matches the initiating service:
-
-image::./images/dt-sampling-example.png[How sampling impacts distributed tracing]
+Regardless of the above, cost conscious customers are likely to be fine with a lower sample rate.
```

Comment:
Please double-check that these are good examples

Reply:
Looks great. Maybe also mention outcome? i.e. failed operations may be of more interest than successful ones.